src/main/scala/org/broadinstitute/dsde/firecloud/service/LibraryServiceSupport.scala

```diff
@@ -64,7 +95,12 @@ trait LibraryServiceSupport {
     ontologyDAO.search(leafId) map {
       case Some(terms) if terms.nonEmpty =>
         terms.head.parents.getOrElse(Seq.empty)
-      case None => Seq.empty
+      case None => Seq.empty[TermParent]
+    } recoverWith {
+      case ex:Exception => {
+        logger.warn(s"exception getting term parents from ontology: ${ex.getMessage}")
```

Comment:
The recover handles any exception from `ontologyDAO.search(leafId)` which is not the just the parents, but the full ontology term. Should this be `logger.warn(s"exception getting term from ontology: ${ex.getMessage}")` ?